hardhat.config.ts

```diff
@@ -112,10 +112,26 @@ const config: HardhatUserConfig = {
     localhost: {
       url: 'http://localhost:8545',
```

Comment:
let's change this to be `http://geth-swap.localhost` by default, since we will be using it against the `beelocal` cluster